api/docs/developer-system-initialization.md

```diff
@@ -0,0 +1,37 @@
+# Developer System Initialization
+## Gcloud Setup
```

Comment:
Link to the other one with text something like "prerequisite: system requirements"